README.md

```diff
@@ -245,6 +247,39 @@ The following properties are configured in the server configuration file(waggle-
 |:----|:----:|:----|
 | `overwrite-config-on-shutdown`    | No         | Controls whether the federations configuration must be overwritten when the server is stopped. Settings this to `false` will cause any federations dynamically added at runtime to be lost when the server is stopped. This is also the case of databases created at runtime when `database-resolution` is set to `MANUAL`. Default is `true`. |
 
+#### Federate to AWS Glue Catalog
+
+Waggle Dance supports federation to AWS Glue Catalog. The federation only works as read-only. Write (Create/Alter/Drop) operations are not supported very well as the Glue APIS don't expose all Hive Metastore functions for instance lock/transactions and other functions are not supported so clients might get exceptions when using certain operations (this can depend on a client like Hive, Spark, etc...). Some research has been done to allow write operations and it is not impossible with a bit more work but out of scope at the moment.
+The GlueConfig configuration should be used if federation to Glue is needed.
+
+
+| Property        | Required | Description |
+|:----|:----:|:----|
+| `glue-account-id` | Yes (if glueConfig used)   | The AWS account number.|
+| `glue-endpoint`   | Yes (if glueConfig used)   | The AWS glue endpoint. Is the same for all accounts key, is the region as Glue catalogs are per account and per region.|
```

Comment:
```suggestion
| `glue-endpoint`   | Yes (if `glueConfig` used)   | The AWS glue endpoint. Is the same for all accounts key, is the region as Glue catalogs are per account and per region.|
```

Reply:
> Is the same for all accounts key, is the region as Glue catalogs are per account and per region.

I'm not sure what this means.  Can we clarify?

Reply:
brainfarth, will make that more clear